Lib/test/test_turtle.py

```diff
@@ -526,6 +546,118 @@ def test_save(self) -> None:
             with open(file_path) as f:
                 assert f.read() == "postscript"
 
+    def test_no_animation_sets_tracer_0(self):
+        s = turtle.TurtleScreen(cv=unittest.mock.MagicMock())
+
+        with s.no_animation():
+            assert s.tracer() == 0
+
+    def test_no_animation_resets_tracer_to_old_value(self):
+        s = turtle.TurtleScreen(cv=unittest.mock.MagicMock())
+
+        for tracer in [0, 1, 5]:
+            s.tracer(tracer)
+            with s.no_animation():
+                pass
+            assert s.tracer() == tracer
+
+    def test_no_animation_calls_update_at_exit(self):
+        s = turtle.TurtleScreen(cv=unittest.mock.MagicMock())
+        s.update = unittest.mock.MagicMock()
+
+        with s.no_animation():
+            s.update.assert_not_called()
+        s.update.assert_called_once()
+
+
+class TestTurtle(unittest.TestCase):
+    def test_begin_end_fill(self):
+        """begin_fill and end_fill counter each other."""
```

Comment:
Docstrings in test methods clutter the test run report when run with `-v` (verbose mode). I'd suggest to either convert these to normal comments `# ...`, or leave them out if they are not needed.

Reply:
Addressed with 28a5ac6d87b6575ff2ef0350aa343d5ca35e7229